### DISCUSSION TITLE
Recording start position is 2.89, 3.38 for live instruments

### DISCUSSION THREAD

**Discussion by jimmydeer** [helgoboss/helgobox#1129] [Q&A]
I was trying to move the clip start/lengths around for the best take in a long clip, but the starts are all at odd times. Is this due to PDC? They playback in time. This is a guitar part:
![Screenshot 2024-08-25 at 9 18 00 PM](https://github.com/user-attachments/assets/9301dc2a-5a6c-4114-a0fd-243b1b718109)

**Comment by helgoboss** [comment]
This is not due to PDC. Are you using the "Static section" or the "Dynamic section"? LOL, your screenshots are always a bit too small, so I miss important context info ;) 

For now, I just assume you used the "Static section". In that case it's probably because that section was already set by Playtime itself after recording. That means the start position was not at 0 beats when you started adjusting it, but e.g. at 0.38 beats. And then when you click-and-drag, you can easily end up at 3.38 beats.

You have multiple options:

- **Recommended:** Use the dynamic section instead (which should initially be at 0.00 beats, so click-and-drag should result in even beats without decimal places)
- Or keep using the static section end right-click the text field to enter a specific number. But you must be super careful with this. If you just enter 4, things will probably be out of time because you didn't take the initial start position into account.

I think I will hide the static section and put it into an "expert menu". Too many users already struggled with the "two-section" concept. I created issue #1132 for this.

**Comment by jimmydeer** [comment]
Hi, I didn't do any moving, that's what the static section was after
recording.

I started a new song. Here's what it says immediately after recording, I
didn't touch anything-added XO, hit start, recorded after 1 bar. no other
plugins

[image: Screenshot 2024-08-26 at 3.45.06 AM.png]

Consequently, I guess I already forgot how to set the clip length-how do I
do that before starting recording?

Thanks

**Reply by helgoboss** [reply]
Can't see your screenshot, but I think you just want to show me that the static section contains a start position > 0 already after recording? It's supposed to be like this. Playtime always records everything, even material in the count-in phase (just like REAPER does with punch-in recording). And then it uses the static section to get the correct cut.

You really want to use the dynamic section instead (and I will hide the static section into an expert section, as this proves once again that it confuses people).

Changing clip length before recording: Click matrix column header => Global recording settings => Enable  fixed-length recording  / set length

**Comment by jimmydeer** [comment]
Understood,  thanks! Keep up the good work
